multiwfn/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, lib, makeWrapper, requireFile, autoPatchelfHook, unzip, writeScriptBin, fetchurl
+, bash
+, glibc
+, xorg
+, libGL
+, motif
+}:
+stdenv.mkDerivation rec {
+    pname = "multiwfn";
+    version = "3.7";
+
+    nativeBuildInputs = [
+      makeWrapper
+      autoPatchelfHook
+      unzip
+    ];
+
+    buildInputs = [
+      glibc
```

Comment:
Do we really need `glibc` in the build inputs for the package to work?

Reply:
Actually not. Have removed it